src/index.html

```diff
@@ -4,13 +4,24 @@
         <meta charset="UTF-8" />
         <title>Nova: text-based programming for beginners</title>
     </head>
-
+    <script src="https://cdn.jsdelivr.net/pyodide/v0.18.0/full/pyodide.js"></script>
```

Comment:
If this cdn ever becomes unavailable will need to find some other pyodide distribution.

Reply:
this is highly available, I'm not really worried about it. if one server fails it will be delivered from another server from jsdeliver.

Reply:
we can serve it from localhost at some point as well.